javalib-ex/src/main/scala/java/io/DataInputStream.scala

```diff
@@ -69,24 +54,27 @@ class DataInputStream(in: InputStream) extends FilterInputStream(in)
   }
 
   def readChar(): Char = {
-    if (hasArrayBuffer)
+    if (hasArrayBuffer) {
       bufDataView.getUint16(consumePos(2)).toChar
-    else
-      view(2).getUint16(0).toChar
+    } else {
+      ((readByte() << 8) | (readByte() & 0xff)).toChar
+    }
```

Comment:
There's no need to add those braces, since both then and else parts are single-line. The same applies in other `readXYZ()` methods.